Guided Investigation - WAF data.ipynb

```diff
@@ -0,0 +1,4156 @@
+{
```

Comment:
<p>Not very clear what this is meant to show you - esp since all col values are zero. Are there some not visible that are none-zero? Might be good to filter down/group and show those.</p><p>In any case, I think a short explanation of what you are seeing here would be good (even if you can infer from previous discussion).</p><p>Maybe just add a title line </p><p>md("Table of X showing Y", "bold")</p><p> </p><p>md is a shortcut function for display(HTML(text)) - it gets imported if you use nbinit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Meant to show a sample of the TFIDF table used to create the Balloon plots subsequently - But since I am displaying the plots a cell later, will eliminate this table.</p>